Makefile

```diff
@@ -55,6 +55,14 @@ release: deps windows_build darwin_build linux_build bsd_build 5_byte_linux_buil
 	$(call build_large,windows,amd64,.exe)
 	$(call zip_large,windows,amd64,.exe)
 
+5_byte_arm_build: $(sources)
```

Comment:
32bit arm may not be that useful

Reply:
I've spent the morning with google trying to figure out why, but I don't think I have. :(
My guess is either:

- something with bit-shifting by 32 being undefined on 32-bit
- `[]byte` can only hold 4 bytes on 32-bit

I'm guessing large disk weed will work until it needs to use that 5th byte and then it'll panic or worse.